app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
 
     <application
         android:name=".MyApplication"
+        android:largeHeap="true"
```

Comment:
@anantprsd5 this is just a hack around that lets you app use more memory.
A better approach to this issue is to cache the image instead.

Please check the following link for more info on this, 
https://developer.android.com/topic/performance/graphics/cache-bitmap.html